mount/mount.go

```diff
@@ -2,66 +2,29 @@
 
 package mount
 
-import (
-	"fmt"
-	"sort"
-
-	"github.com/moby/sys/mountinfo"
-)
-
 // Mount will mount filesystem according to the specified configuration.
 // Options must be specified like the mount or fstab unix commands:
 // "opt1=val1,opt2=val2". See flags.go for supported option flags.
+//
+// Mount is not implemented on Windows.
 func Mount(device, target, mType, options string) error {
 	flag, data := parseOptions(options)
 	return mount(device, target, mType, uintptr(flag), data)
 }
 
 // Unmount lazily unmounts a filesystem on supported platforms, otherwise does
-// a normal unmount.  If target is not a mount point, no error is returned.
+// a normal unmount. If target is not a mount point, no error is returned.
+//
+// Unmount is not implemented on Windows.
 func Unmount(target string) error {
 	return unmount(target, mntDetach)
 }
 
 // RecursiveUnmount unmounts the target and all mounts underneath, starting
 // with the deepest mount first. The argument does not have to be a mount
 // point itself.
+//
+// RecursiveUnmount is not implemented on Windows.
 func RecursiveUnmount(target string) error {
```

Comment:
So, alternatively, I guess we could not implement at all on Windows (move this to a `_linux.go` or `_unix.go` wdyt?

Reply:
Yeah, I think the rename + build flags will be slightly cleaner.